elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/AsyncProperties.java

```diff
@@ -33,6 +33,11 @@
      */
     private int queryCleanupDays = 7;
 
+    /**
+     * Default cancel interval of async query.
+     */
+    private int queryCancellationSeconds = 300;
```

Comment:
queryCancellationIntervalSeconds

Reply:
I want to keep it similar to cleanup. cleanup  is called queryCleanupDays. So queryCancellationSeconds sounds correct to me.